packages/garbo/src/tasks/post/index.ts

```diff
@@ -373,11 +373,21 @@ function acquireAbortFreeRun(): GarboPostTask {
   };
 }
 
+function uneffectAttunement(): GarboPostTask {
+  return {
+    name: "Uneffect Eldritch Attunement After 11 Tentacles",
+    ready: () => get("_eldritchTentaclesFoughtToday") >= 11,
+    completed: () => !have($effect`Eldritch Attunement`),
+    do: () => uneffect($effect`Eldritch Attunement`),
```

Comment:
Is it worth adding an acquire step for sgeea?

Reply:
I think right now we'd either abort or hit an infinite loop, right?

I know the price cap I outlined in discord:
 ```typescript
haveEffect($effect`Eldritch Attunement`) * get("valueOfAdventure")
```
is probably economically not right, but I think having an abort is strictly better than the silent failure.

Reply:
maybe we could clamp that with some sane bounds?